app/portfolio/page.tsx

```
@@ -0,0 +1,353 @@
+    return () => clearTimeout(t);
+  }, []);
+
+  if (isLoading) {
+    return <LoadingScreen onComplete={() => setIsLoading(false)} />;
+  }
+
+  return (
+    <div className="relative w-full bg-black text-white">
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)